spec/support/api/schemas/song.json

```diff
@@ -76,6 +76,10 @@
         "formatted_duration": {
           "id": "http://spotify-apprentice-app.com/song/title",
           "type": "string"
+        },
+        "genre_id": {
+          "id": "http://spotify-apprentice-app.com/song/title",
+          "type": "string"
```

Comment:
If genre is not required, this should also be able to be null: `["string", "null"]`